tribler.spec

```diff
@@ -93,18 +93,18 @@ hiddenimports = [
     'csv',
     'dataclasses',  # https://github.com/pyinstaller/pyinstaller/issues/5432
     'ecdsa',
-    'pyaes',
     'PIL',
-    'scrypt', '_scrypt',
-    'sqlalchemy', 'sqlalchemy.ext.baked', 'sqlalchemy.ext.declarative',
     'pkg_resources', # 'pkg_resources.py2_warn', # Workaround PyInstaller & SetupTools, https://github.com/pypa/setuptools/issues/1963
-    'requests',
+    'pyaes',
     'pydantic',
-    'PyQt5.QtTest',
     'pyqtgraph',
     'pyqtgraph.graphicsItems.PlotItem.plotConfigTemplate_pyqt5',
     'pyqtgraph.graphicsItems.ViewBox.axisCtrlTemplate_pyqt5',
     'pyqtgraph.imageview.ImageViewTemplate_pyqt5',
+    'PyQt5.QtTest',
+    'requests',
+    'scrypt', '_scrypt',
```

Comment:
I believe `pyaes`, `sqlalchemy` and `script` are not required anymore. These were dependencies for `bitcoinlib` (yes, we used to have a Bitcoin wallet in Tribler 😁 ) but we remove that library and we apparently forgot to remove them from `tribler.spec`.

Reply:
Good to know! I'll remove them in a separate PR after additional testing